_sass/blocks/_revenues_subpage_steps.scss

```diff
@@ -0,0 +1,125 @@
+// Revenues Subpage Steps
+//
+// Diving Graphics and associated steps
+//
+// Styleguide blocks.revenues_subpage_steps
+
+$onshore-oil-primary-color: #534136;
+$onshore-oil-secondary-color: #c5403c;
+$offshore-oil-primary-color: #b3e4f9;
+$offshore-oil-secondary-color: $blue;
+$coal-primary-color: #534136;
+$coal-secondary-color: #c5403c;
+$minerals-primary-color: #534136;
+$minerals-secondary-color: #c5403c;
+$offshore-renewables-primary-color: #534136;
+$offshore-renewables-secondary-color: #0b92ce;
+$onshore-renewables-primary-color: #579469;
+$onshore-renewables-secondary-color: $blue;
+
+@import 'revenues_subpage_diving_graphics';
+
+.revenues_subpage-dyk {
+  position: relative;
+  border-radius: $base-border-radius;
+  margin-bottom: $base-padding;
+  overflow: hidden;
+
+  @include respond-to(medium-up) {
+    display: block;
+    margin-top: 40%;
+    @include span-columns(3);
+  }
+}
+
+.revenues_subpage-dyk_heading {
```

Comment:
Selector `revenues_subpage-dyk_heading` should be written in lowercase with hyphens